src/components/sets/SetsManager.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Tabs } from 'antd';
+import Tabs from 'antd/es/tabs';
```

Comment:
What does this change do? (Just link to relevant docs, if that's easiest.)

Reply:
I thought it would reduce the bundle size but it turns out that is not true, so I have reverted the change. It did however prevent the other ant-design component files from appearing in the firefox debugger.